writer.go

```diff
@@ -76,3 +76,33 @@ func open(paths []string) ([]zapcore.WriteSyncer, func(), error) {
 	}
 	return writers, close, errs.AsError()
 }
+
+// OpenWriters combines the passed io.Writer objects that are compatible with
+// zapcore.WriteSyncer into a locked WriteSyncer. It returns any errors
+// encountered.
+func OpenWriters(writers ...io.Writer) (zapcore.WriteSyncer, error) {
```

Comment:
Agreed. In cases where this is needed, I'd much prefer to have users deal with it explicitly.

Please remove this function.